### DISCUSSION TITLE
Iptv hindi channel doesn't work

### DISCUSSION THREAD

**Discussion by Hfhshd** [iptv-org/community#174] [Q&A] [answered]
Majority of the hindi chnnels are not working.
Please update  fix and add channels 
Zee 
Sab 
Star plus
Sony

**Comment by Achu2234** [comment, marked as answer]
Star plus is made by star group so all links star group is blocked
then u can use https://pubads.g.doubleclick.net/ssai/event/HgaB-u6rSpGx3mo4Xu3sLw/master.m3u8 this link for sony
this link can use for zee 5 https://d3t34m8y8rh0vb.cloudfront.net/preview/zeekannadahd/800.m3u8

**Comment by BellezaEmporium** [comment]
API = https://catalogapi.zee5.com/v1/channel/ *channel location on Zee5*
Token = https://useraction.zee5.com/token/live.php

How to get them to work ?

-> Take the channel you want from the Zee5 Live TV (we'll say &Flix as an example)
-> Take this part of the URL ![image](https://user-images.githubusercontent.com/30985701/141470271-5da831a2-f91e-4e24-ad62-4e089cd2cb2f.png)
-> Paste it at the end of the API link, and you'll get all the results.
-> Scrape the stream URL, query the Token website and paste the result at the end of the stream
-> Enjoy !
![image](https://user-images.githubusercontent.com/30985701/141470612-4c7cf827-4617-4002-a4c5-8272a8d033a7.png)

**Reply by BellezaEmporium** [reply]
Test : https://iptv-all.lanesh4d0w.repl.co/india/zee_all_in_1?channel=0-9-channel_2105335046
